diff -up ./addons/pvr.iptvsimple/src/PVRIptvData.cpp.orig ./addons/pvr.iptvsimple/src/PVRIptvData.cpp
--- ./addons/pvr.iptvsimple/src/PVRIptvData.cpp.orig	2014-03-14 11:52:32.000000000 +0100
+++ ./addons/pvr.iptvsimple/src/PVRIptvData.cpp	2014-03-24 03:23:17.168000000 +0100
@@ -27,7 +27,7 @@
 #include <fstream>
 #include <map>
 #include "zlib.h"
-#include "rapidxml/rapidxml.hpp"
+#include "rapidxml.h"
 #include "PVRIptvData.h"
 
 #define M3U_START_MARKER        "#EXTM3U"
@@ -1004,4 +1004,4 @@ int PVRIptvData::GetChannelId(const char
     iId = ((iId << 5) + iId) + c; /* iId * 33 + c */
 
   return abs(iId);
-}
\ Pas de fin de ligne à la fin du fichier
+}
